eConsciencia/src/App.jsx

```diff
@@ -24,14 +26,15 @@ const App = () => {
         <Route path="/" element={<AuthComponent />} />
         <Route path="/signup" element={<SignUpForm />} />
         <Route path="/signin" element={<AuthComponent />} />
+        <Route path="/quiz-shortage" element={<QuizShortage />} />
```

Comment:
remove path to quiz shortage, cuz there is only one quiz if you need to render QuizShortage use QuizMian